q2_metadata/normalization/_norm_errors.py

```diff
@@ -0,0 +1,204 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import yaml
+
+
+class Error(Exception):
+    """Base class for exceptions encountered
+    in the rules checking."""
+
+    @classmethod
```

Comment:
I don't think this needs to be a classmethod, right?

Reply:
right